### PR TITLE
[4.0] PackageAdapter and Updater parameterized queries

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Adds some missing parameterized queries to `Joomla\CMS\Installer\PackageAdapter` and `Joomla\CMS\Updater\Updater`.

### Testing Instructions

Simple queries, code review is fine.

### Documentation Changes Required

No.

### CODE REVIEW

File: libraries/src/Installer/Adapter/PackageAdapter.php [hunk 1]
```diff
@@ -316,8 +317,8 @@ protected function finaliseUninstall(): bool
 
 		// Remove the schema version
 		$query = $db->getQuery(true)
-			->delete('#__schemas')
-			->where('extension_id = :extension_id')
+			->delete($db->quoteName('#__schemas'))
+			->where($db->quoteName('extension_id') . ' = :extension_id')
 			->bind(':extension_id', $extensionId, ParameterType::INTEGER);
```
Comment (Quy) [inline libraries/src/Installer/Adapter/PackageAdapter.php]: Please consider removing the assignment on line 314 and use it directly like you did on line 231.

Comment (SharkyKZ) [reply libraries/src/Installer/Adapter/PackageAdapter.php]: Done.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 2633260257294efe9d86803847dba8455f62aa8c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27105">issues.joomla.org/tracker/joomla-cms/27105</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**